elm/README.md

```diff
@@ -20,3 +20,37 @@ In the private area, you'll see the contents for the logged in user and you can
 
 - add new contents with their tags;
 - filter the shown contents by tag.
+
+## Specification
+
+The `spec` folder contains some end-to-end acceptance tests written using [Quickstrom](https://quickstrom.io/).
+
+To run them, you could execute the following commands inside the `spec` folder, given your application is exposed on `localhost:8000`:
+
+```
+docker run --rm -d \
```

Comment:
Can we get scripts for these commands (possibly in the `bin/test` directory) ? I don't want users to have to copy/paste these commands manually.

Reply:
sure, makes a lot of sense